api/src/main/kotlin/edu/wgu/osmt/security/SecurityConfig.kt

```diff
@@ -23,14 +23,14 @@ import edu.wgu.osmt.RoutePaths.SKILL_UPDATE
 import edu.wgu.osmt.RoutePaths.TASK_DETAIL_BATCH
 import edu.wgu.osmt.RoutePaths.TASK_DETAIL_SKILLS
 import edu.wgu.osmt.RoutePaths.TASK_DETAIL_TEXT
-import edu.wgu.osmt.RoutePaths.createUuidRegex
```

Comment:
If we are removing the import here, is the method still used anywhere? I'd say probably not. Can we remove it?